src/Core/Utils.fs

```diff
@@ -56,8 +56,11 @@ module Option =
 
 [<RequireQualifiedAccess>]
 module Document =
-    let (|FSharp|Other|) (document : TextDocument) =
-        if document.languageId = "fsharp" then FSharp else Other
+    let (|FSharp|CSharp|VB|Other|) (document : TextDocument) =
```

Comment:
I don't think I care enough to create separate plugin for it. ;)

(It may change with introducing windows debugger since it also works for C# and VB)